index.html

```diff
@@ -1330,8 +1330,10 @@ <h2>DID Subject</h2>
 such as when a <a>DID resolver</a> is performing <a>DID resolution</a>.
 However, the fully resolved <a>DID document</a> always contains a valid
 <code><a>id</a></code> property. The value of <code><a>id</a></code> in the
-resolved <a>DID document</a> is expected to match the <a>DID</a> that was
-resolved.
+resolved <a>DID document</a> MUST match the <a>DID</a> that was
+resolved, or be populated by the <a>DID method</a> with another <a>DID</a> it 
```

Comment:
I'm good with this. The "or..." statement seems like it could cause some tricky issues with how de-referencing would work on identifying the did document, such as sub-objects (verificationMethod block, service block, etc) being identified with a different base did than the top level id. However, since the point of this statement is to point out that the method authors must state how it's canonically mapped, I think it should work out fine as long as the method is written properly.